doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -78,7 +78,7 @@ For lists of protocol-specific changes, see `Protocols`_.
 Bluetooth samples
 -----------------
 
-* |no_changes_yet_note|
+* :ref`direct_test_mode` and :ref:`radio_test` - Added support for front-end module devices that support 2-pin PA/LNA interface with additional support for the Skyworks SKY66114-11 and the Skyworks SKY66403-11.
```

Comment:
A nitpick: Radio test sample is not exactly a Bluetooth sample, so it would require an entry under Other samples.

Reply:
Good catch. Fixed